contentcuration/contentcuration/frontend/channelEdit/views/trash/__tests__/trashModal.spec.js

```diff
@@ -134,7 +134,8 @@ describe('trashModal', () => {
 
     it('clicking CLOSE on delete confirmation dialog should close the dialog', async () => {
       await wrapper.setData({ showConfirmationDialog: true });
-      await wrapper.findComponent('[data-test="closeconfirm"]').trigger('click');
+      const modal = wrapper.findComponent({ name: 'KModal' });
```

Comment:
Can we use the `data-test` property to specifically target the `KModal` in question.